integration/spark2/src/main/scala/org/apache/spark/sql/parser/CarbonSpark2SqlParser.scala

```diff
@@ -247,7 +248,7 @@ class CarbonSpark2SqlParser extends CarbonDDLSqlParser {
         val unresolvedrelation =
         CarbonReflectionUtils.getUnresolvedRelation(
           tableIdentifier,
-          SparkSession.getActiveSession.get.version,
+          SPARK_VERSION,
```

Comment:
can you remove this parameter and do if check inside `CarbonReflectionUtils.getUnresolvedRelation`